[IT] Update `Main_page` (#9627)
ppy/osu-wiki: wiki/Main_page/it.md

Le riposte a domande frequenti e le soluzioni per problemi comuni si trovano qui.

[Centro aiuto](/wiki/Help_centre) • [Risoluzione problemi di performance](/wiki/Performance_troubleshooting) • [Team di supporto account](/wiki/People/Account_support_team) • [Forum d'aiuto](https://osu.ppy.sh/forum/5)

Restrizioni del profilo: [Silenziamento](/wiki/Silence) • [Restrizioni account](/wiki/Help_centre/Account_restrictions)

> Fai la tua parte: [Come e in cosa aiutare](/wiki/Community/How_you_can_help!) • [Segnala comportamenti scorretti](/wiki/Reporting_bad_behaviour) • [Segnala abuso](/wiki/Reporting_bad_behaviour/Abuse) • [Segnala gli imbroglioni](/wiki/Reporting_bad_behaviour/Handling_foul_play)

Reviewer: ```suggestion
Fai la tua parte: [Come e in cosa aiutare](/wiki/Community/How_you_can_help!) • [Segnala comportamenti scorretti](/wiki/Reporting_bad_behaviour) • [Segnala abuso](/wiki/Reporting_bad_behaviour/Abuse) • [Segnala i cheater](/wiki/Reporting_bad_behaviour/Handling_foul_play)
```

Le interazioni sociali sono una delle cose principali che alimentano i giocatori di osu! in tutto il mondo. Esplora le attività e i concorsi, entra in contatto con altri, o aiuta ad organizzare il prossimo grande evento!

Posti: [Forum](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat) • [Server Discord](/wiki/Community/Discord_servers)

[Tornei](/wiki/Tournaments) • [Beatmap Spotlights](/wiki/Beatmap_Spotlights) • [Concorsi](/wiki/Contests) • [Programma di mentorship della comunità](/wiki/Community/Community_Mentorship_Program)

> [Projects](/wiki/Community/Projects) • [Guilda dei Mapper](/wiki/Community/Mappers_Guild) • [Progetto Amata](/wiki/Community/Project_Loved) • [Meeting della community di osu!](/wiki/Community/osu!_community_meetings) • [osu!academy](/wiki/Community/Video_series/osu!academy) • [osu!mapping](/wiki/Community/Video_series/osu!mapping) • [osu!monthly](/wiki/Community/osu!monthly) • [osu!talk](/wiki/Community/Video_series/osu!talk)

Reviewer: ```suggestion
[Projects](/wiki/Community/Projects) • [Gilda dei Mapper](/wiki/Community/Mappers_Guild) • [Progetto Amata](/wiki/Community/Project_Loved) • [Meeting della community di osu!](/wiki/Community/osu!_community_meetings) • [osu!academy](/wiki/Community/Video_series/osu!academy) • [osu!mapping](/wiki/Community/Video_series/osu!mapping) • [osu!monthly](/wiki/Community/osu!monthly) • [osu!talk](/wiki/Community/Video_series/osu!talk)
```

Come quasi tutto il resto là fuori, la wiki di osu! è scritta e mantenuta da volontari. Suggerisci un miglioramento, segnala un problema, o scrivi un tuo articolo sulla [repository di GitHub di osu!wiki](https://github.com/ppy/osu-wiki).

> [Storia di osu!](/wiki/History_of_osu!) • [Storia del Classifiche Online](/wiki/History_of_osu!/Online_rankings) • [Storia della wiki di osu!](/wiki/History_of_osu!/osu!_wiki) • [Timeline del mapping e del modding](/wiki/History_of_osu!/Mapping_and_modding_timeline) • [Pesci d'aprile](/wiki/History_of_osu!/April_Fools)

Reviewer: ```suggestion
[Storia di osu!](/wiki/History_of_osu!) • [Storia delle Classifiche Online](/wiki/History_of_osu!/Online_rankings) • [Storia della wiki di osu!](/wiki/History_of_osu!/osu!_wiki) • [Timeline del mapping e del modding](/wiki/History_of_osu!/Mapping_and_modding_timeline) • [Pesci d'aprile](/wiki/History_of_osu!/April_Fools)
```
